### PR TITLE
Add sbt-release

### PR DESCRIPTION
**Pull request by changhiskhan**

### CODE REVIEW

File: docs/release.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Release Maven Jar
+-----------------
+We release the rikai jar to the maven artifactory using
+`sbt-sonatype <https://github.com/xerial/sbt-sonatype>`_ in conjunction with
+`sbt-release <https://github.com/sbt/sbt-release>`_. You can follow `these instructions
+<https://www.scala-sbt.org/1.x/docs/Using-Sonatype.html>`_ to setup :code:`sbt-sonatype`.
+
+Sanpshot builds can be published to Sonatype snapshots repo using :code:`sbt publishSigned`.
+Release builds can be published to Sonatype staging repo then sync'd to maven using
+:code:`sbt release`
```
Comment (eddyxu) [inline docs/release.rst:17]: Curious what do we need to do for automatically release in github actions?
Or are we planning to do manual release for the timebeing?

**Review by eddyxu — APPROVED** [review]
LGTM

File: docs/release.rst (hunk 1 above)
Comment (changhiskhan) [reply docs/release.rst]: i didn't find an official integration but some of the actions in the marketplace looked promising

File: python/rikai/__version__.py [hunk 2]
```diff
@@ -11,3 +11,5 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+
+version = "0.0.1.dev0"
```
Comment (eddyxu) [inline python/rikai/__version__.py:15]: this makes import a little bit non nature?
as it need `rikai.__version__.version` to get the string

**Review by eddyxu — COMMENTED** [review]
small nits

File: python/rikai/__version__.py (hunk 2 above)
Comment (changhiskhan) [reply python/rikai/__version__.py:15]: i'll import it from __init__ so you can do `rikai.version`

Comment (eddyxu) [reply python/rikai/__version__.py:15]: Cool, feel free to merge after this

**Pull request merged**